mula/scheduler/schedulers/schedulers/boefje.py

```diff
@@ -685,6 +696,49 @@ def get_oois_for_boefje(self, boefje: models.Plugin, organisation: str) -> list[
 
         return oois
 
+    # TODO:: exception handling
+    def is_ooi_in_other_organisations(self, boefje_task: models.BoefjeTask):
+        """Is the ooi in other organisations? When the ooi is in other
+        organisations we don't want to scan those as well, we still create
+        tasks for those organisations but they will be batched together.
+        """
+        # From the boefje id we can get the same boefje in other organisations.
+        configs = self.ctx.services.katalogus.get_configs(boefje_id=boefje_task.boefje.id, enabled=True)
+        for config in configs:
+            if config.organisation_id == boefje_task.organization:
+                # Skip the organisation we are currently in
+                continue
+
+            ooi = self.ctx.services.octopoes.get_object(config.organisation_id, boefje_task.input_ooi)
```

Comment:
probably need an optimized octopoes bulk call for this

Reply:
https://github.com/minvws/nl-kat-coordination/pull/4459